tsdb/compact_test.go

```diff
@@ -1096,7 +1096,7 @@ func BenchmarkCompactionFromHead(b *testing.B) {
 			opts := DefaultHeadOptions()
 			opts.ChunkRange = 1000
 			opts.ChunkDirRoot = chunkDir
-			h, err := NewHead(nil, nil, nil, opts)
+			h, err := NewHead(nil, nil, nil, 1, opts)
```

Comment:
nit: why 1 exemplar?

Reply:
Even 0 works. This benchmark does not use exemplars.

Reply:
thats kinda my point...